app/scripts/lib/sentry.js

```diff
@@ -48,6 +48,16 @@ define(function (require, exports, module) {
         data.request.url = cleanUpQueryParam(data.request.url);
       }
 
+      if (data.tags) {
+        // if this is a known errno, then use grouping with fingerprints
+        // Docs: https://docs.sentry.io/hosted/learn/rollups/#fallback-grouping
+        if (data.tags.errno) {
+          data.fingerprint = ['errno' + data.tags.errno];
```

Comment:
I tested this by logging the same 2 events with the same stack trace with and without this fingerprint field:

![](http://v14d.com/i/57bc885d3adf3.png)

This event was marked unique when I adjusted the `fingerprint` value:
![image](https://cloud.githubusercontent.com/assets/128755/17902608/f2652da8-6935-11e6-810b-0a093b9259f0.png)